src/api/components/Section/Section.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+type Props = {
+  title?: string;
+  children: React.ReactNode;
+};
+
+export const Section: React.FC<Props> = (
+  {
+    title = '',
+    children,
+  },
+) => {
```

Comment:
Fix all cases please
```suggestion
export const Section: React.FC<Props> = ({
    title = '',
    children,
}) => {
```